presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -979,6 +980,60 @@ core::LocalPartitionNode::Type toLocalExchangeType(
       VELOX_UNSUPPORTED("Unsupported exchange type: {}", toJsonString(type));
   }
 }
+
+class HashPartitionFunctionSpec : public core::PartitionFunctionSpec {
```

Comment:
Are the partition changes related to the http memory allocation? If not can we put them in a separate PR? Thanks!